azure-devops/azext_devops/dev/boards/work_item.py

```diff
@@ -199,18 +200,29 @@ def _handle_vsts_service_error(ex):
     raise CLIError(ex)
 
 
-def show_work_item(id, open=False, organization=None, detect=None):  # pylint: disable=redefined-builtin
+def show_work_item(id, as_of=None, expand=None, fields=None, open=False, organization=None, detect=None):  # pylint: disable=redefined-builtin
     """Show details for a work item.
     :param id: The ID of the work item
     :type id: int
+    :param as_of: AsOf date time string Example: '2019-01-20 00:20:00'
+    :type as_of:string
+    :param expand: The expand parameters for work item attributes.
+    :type expand:str
+    :param fields: Comma-separated list of requested fields. Example:System.Id,System.AreaPath
```

Comment:
maybe point to some doc explaining what are the possible values will be helpful
cc: @geverghe

Reply:
This is with respect to the fields parameter? Yes, docs would be useful to explain this.

Reply:
yes for `param fields`

Reply:
Adding a link to API for fetching fields
https://docs.microsoft.com/en-us/rest/api/azure/devops/wit/fields/list?view=azure-devops-rest-5.0